modules/ROOT/pages/api-groups-creating-groups.adoc

```diff
@@ -0,0 +1,80 @@
+= Create API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can bundle a set of APIs to enable ease-of-use and efficiency for your API consumers. To create a bundle, you first create an API Group and then add the relevant API instances to it. If you want your API Group to be discovered by consumers, you must publish it to Anypoint Exchange.
+
+You can create API Groups using the following steps:
+
+*Step 1:* <<create-api-groups,Create an API Group>>
+
+The first step to creating an API Group that can be used by consumers, is create an API Group. 
+
+*Step 2:* <<adding-api-instances-to-groups,Add API instances to the new API Group>>
+
+After you create the API Group, you then add API instances to this group for the first time.
+
+== Task Prerequisites
+
+Before you begin, you must have:
+
+* At least one API instance in your organization or business group.
+* API Group Administrator permissions for all the business groups and environments in which your API instances are located. 
++
+Otherwise, you may not find the required organization or business group in the list.
+
+[[create-api-groups]]
+== Create API Groups
+
+You can create an API Group for a specific organization and then add this group to multiple organizations, if required.
```

Comment:
The group is created in an organization and it cannot be added to multiple organizations. We will be able to add group instances to the environments of the organization in which you have created the group instance

Reply:
Done! Thanks!